packages/login/src/modules/login/reducer.js

```diff
@@ -0,0 +1,43 @@
+import * as actions from './actions'
+import {Pages} from './../../types/Pages'
+
```

Comment:
functions could be written as arrow functions (in order to maintain consistency with other reducers).